scripts/certificates

```diff
@@ -127,7 +115,7 @@ certificate_log "info"  "Creating a self signed SSL certificate ..." "outonly" "
 # Add more info /C=$country/ST=$state/L=$locality/O=$organization/OU=$organizational_unit/CN=$common_name/emailAddress=$email
 SUBJ="/C=${SSL_COUNTRY}/ST=${SSL_STATE}/O=${SSL_ORGANIZATION}/CN=${SSL_NAME}/emailAddress=${SSL_EMAIL}"
 
-openssl genpkey -out "${SSL_LOCATION}/fledge.pass.key" -pass pass:fledge -algorithm RSA 2> /dev/null
+openssl genpkey -out "${SSL_LOCATION}/fledge.pass.key" -pass pass:fledge -algorithm RSA -pkeyopt rsa_keygen_bits:${KEY_SIZE} 2> /dev/null
 openssl rsa -passin pass:fledge -in "${SSL_LOCATION}/fledge.pass.key" -out "${SSL_LOCATION}/${SSL_NAME}.key" 2> /dev/null
 rm "${SSL_LOCATION}/fledge.pass.key"
 openssl req -new -key "${SSL_LOCATION}/${SSL_NAME}.key" -subj "${SUBJ}" -out "${SSL_LOCATION}/${SSL_NAME}.csr" 2> /dev/null
```

Comment:
We should delete the CSR after usage?

Reply:
The certs API has utilized this, therefore I am uncertain regarding the deletion.